lib/Dialect/TTIR/IR/TTIROps.cpp

```diff
@@ -1012,6 +1012,36 @@ ::mlir::LogicalResult mlir::tt::ttir::AllGatherOp::verify() {
   return success();
 }
 
+//===----------------------------------------------------------------------===//
+// ReverseOp
+//===----------------------------------------------------------------------===//
+
+::mlir::LogicalResult mlir::tt::ttir::ReverseOp::verify() {
+  auto dimensions = getDimensions();
+  llvm::SmallDenseSet<int64_t> uniqueDims(dimensions.begin(), dimensions.end());
+
+  if (uniqueDims.size() != dimensions.size()) {
```

Comment:
For each failure, can you please add a negative TTIR Dialect compiler test under:
/test/ttmlir/Dialect/TTIR